### DISCUSSION TITLE
Storage way higher than 20GB

### DISCUSSION THREAD

**Discussion by pepa65** [OffchainLabs/prysm#8816] [Q&A] [answered]
I set my Prysm up according to the official startup guide, and run it with: `./prysm.sh beacon-chain --datadir=/data/Prysm --http-web3provider=/data/Prysm/geth.ipc --fallback-web3provider=https://mainnet.infura.io/v3/7a1fb03bad834ad48f3eb26bfd151d7a --accept-terms-of-use`
Yet the storage has quickly swelled to 245 GB and it's still growing a bit each day. What is happening?? The guide said that you needed 20GB of free space.

**Comment by nisdas** [comment]
@pepa65 Something is very wrong in that case, it should be about 14 GB as of mainnet right now. Are you sure all that is from prysm ? Any chance you might be counting from geth here.  Any other flags you were running previously ?

**Comment by pepa65** [comment, marked as answer]
Geth is in a separate directory, around 176GB at the moment... But..! Just looking at the directory tree inside `/data/Prysm`, there is a directory `geth` that is around 235GB, so that should account for the difference..! No idea how it got there, perhaps ran geth with the wrong `--datadir`..?
Thanks, solved I would say!

**Comment by Jyoti-singh18** [comment]
what is the current size of beacon chain?
